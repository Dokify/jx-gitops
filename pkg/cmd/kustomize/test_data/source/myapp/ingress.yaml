apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: nginx
  name: myapp
  namespace: myapps
spec:
  rules:
  - host: myapp.cluster.local
    http:
      paths:
      - backend:
          serviceName: myapp
          servicePort: 80
