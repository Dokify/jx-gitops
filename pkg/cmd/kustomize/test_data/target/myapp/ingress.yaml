apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: nginx
  name: myapp
  namespace: myapps
spec:
  rules:
  - host: myapp.1.2.3.4.nipio
    http:
      paths:
      - backend:
          serviceName: myapp
          servicePort: 80
