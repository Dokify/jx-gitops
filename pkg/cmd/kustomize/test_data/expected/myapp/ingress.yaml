apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: myapp
  namespace: myapps
spec:
  rules:
  - host: myapp.1.2.3.4.nipio
